_shared/hooks/post_gen_project.py

```diff
@@ -66,6 +80,40 @@ def remove_conditional_files():
                 rmdir(path)
 
 
+def remove_project_files(target_dir):
+    """
+    Remove certain files from the project.
+
+    When running `make template` to update a project with the latest changes
+    from the cookiecutter the cookiecutter includes the ability to remove
+    certain files from the project.
+
+    For example our cookiecutter used to create a tests/pyproject.toml file in
+    projects containing a special pylint config to be used when linting the
+    tests. After we moved away from pylint this tests/pyproject.toml file was
+    no longer needed.  It was helpful to have `make template` automate removing
+    the file.
+
+    This feature should be using sparingly because it's not generally safe: it
+    could remove files that contain uncommitted changes.
+
+    """
+    paths_to_remove = [
+        ".cookiecutter/setuptools/install_requires",
+        ".cookiecutter/setuptools/console_scripts",
+        ".cookiecutter/setuptools/entry_points",
+    ]
```

Comment:
Old includes that've now been deleted from the `setup.cfg` template. This will automatically remove these includes from any projects that were using them. Those projects will now need to change to the new includes from the `pyproject.toml` template.